source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRToggleInputDataSource.mm

```diff
@@ -25,13 +25,17 @@ @implementation ACRToggleInputDataSource
 
 - (instancetype)initWithInputToggle:(std::shared_ptr<ToggleInput> const&)toggleInput
       WithHostConfig:(std::shared_ptr<HostConfig> const&)hostConfig
+       withTintColor:(UIColor*) color;
```

Comment:
I suggest to add getter for _toggleSwitch then modifying the interface. that way developers will have more control over what they can customize.